### PR TITLE
Update installing_selenium_libraries.es.md

### PR DESCRIPTION
**Pull request by AlexAndradeNet**
Translation to Spanish.

I've agreed the CLA

### CODE REVIEW

File: docs_source_files/content/selenium_installation/installing_selenium_libraries.es.md [hunk 1]
```diff
@@ -3,18 +3,14 @@ title: "Instalación de las librerías Selenium"
 weight: 1
 ---
 
-{{% notice info %}}
-<i class="fas fa-language"></i> Page being translated from 
-English to Spanish. Do you speak Spanish? Help us to translate
-it by sending us pull requests!
-{{% /notice %}}
 
 First you need to install the Selenium bindings for your automation project.
```
Comment (diemol) [inline docs_source_files/content/selenium_installation/installing_selenium_libraries.es.md]: There is still some text in English, can you please remove it?

**Review by diemol — CHANGES_REQUESTED** [review]

**Review by AlexAndradeNet — COMMENTED** [review]
Was remove some paragraphs that still remained in English

File: docs_source_files/content/selenium_installation/installing_selenium_libraries.es.md (hunk 1 above)
Comment (AlexAndradeNet) [reply docs_source_files/content/selenium_installation/installing_selenium_libraries.es.md]: Was remove some paragraphs that still remained in English

**Review by diemol — APPROVED** [review]
Thank you @AlexAndradeSan!

**Pull request merged**